numpy/_core/getlimits.py

```diff
@@ -150,6 +150,9 @@ def _float_to_str(self, value):
 #
 # 20230201 - use (ftype(-1.0) / ftype(10.0)).newbyteorder('<').tobytes()
 #            instead because stold may have deficiencies on some platforms.
+# 20340511 - use
+#            v = (np.longdouble(-1.0) / np.longdouble(10.0))
+#            v.view(v.dtype.newbyteorder('<')).tobytes()
```

Comment:
Maybe just rewrite this comment to delete the old versions?

You got the year wrong but IMO keeping the old versions around isn't helpful. I can't make a suggestion including enough context, but something like:

```
# Key to identify the floating point type. Key is result of
# v = (np.longdouble(-1.0) / np.longdouble(10.0))
# v.view(v.dtype.newbyteorder('<')).tobytes()
#
# Uses division to work around deficiencies in strtold on some platforms.
```

Would make this whole point clearer to the reader without getting bogged down in thinking about platform-specific vagaries or numpy API changes.